cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -579,7 +663,7 @@ def test(
     pred_df = keras_model.transform(test_df)
     pred_df.printSchema()
     pred_df.show(5)
-
+    # Now, we write the new predictions as new columns to the output DataFrame.
```

Comment:
Let's not use an instructive tone within the code. It has to be used only before and after the code blocks.

Reply:
I think we can remove this comment since we have it in the next line.